packages/block-library/src/image/edit.js

```diff
@@ -323,371 +276,359 @@ export class ImageEdit extends Component {
+								<ExternalLink href="https://www.w3.org/WAI/tutorials/images/decision-tree">
 									{ __(
-										'(Note: many devices and browsers do not display this text.)'
+										'Describe the purpose of the image'
 									) }
 								</ExternalLink>
+								{ __(
+									'Leave empty if the image is purely decorative.'
+								) }
 							</>
 						}
 					/>
-				</InspectorAdvancedControls>
-			</>
-		);
-
-		// Disable reason: Each block can be selected by clicking on it
-		/* eslint-disable jsx-a11y/click-events-have-key-events */
-		return (
-			<>
-				{ controls }
-				<Block.figure className={ classes }>
-					<ImageSize src={ url } dirtynessTrigger={ align }>
-						{ ( sizes ) => {
-							const {
-								imageWidthWithinContainer,
-								imageHeightWithinContainer,
-								imageWidth,
-								imageHeight,
-							} = sizes;
-
-							const filename = this.getFilename( url );
-							let defaultedAlt;
-							if ( alt ) {
-								defaultedAlt = alt;
-							} else if ( filename ) {
-								defaultedAlt = sprintf(
-									/* translators: %s: file name */
-									__(
-										'This image has an empty alt attribute; its file name is %s'
-									),
-									filename
-								);
-							} else {
-								defaultedAlt = __(
-									'This image has an empty alt attribute'
-								);
-							}
-
-							const img = (
-								// Disable reason: Image itself is not meant to be interactive, but
-								// should direct focus to block.
-								/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
-								<>
-									<img
-										src={ url }
-										alt={ defaultedAlt }
-										onClick={ this.onImageClick }
-										onError={ () =>
-											this.onImageError( url )
-										}
-									/>
-									{ isBlobURL( url ) && <Spinner /> }
-								</>
-								/* eslint-enable jsx-a11y/no-noninteractive-element-interactions */
+					<ImageSizeControl
+						onChangeImage={ updateImage }
+						onChange={ ( value ) => setAttributes( value ) }
+						slug={ sizeSlug }
+						width={ width }
+						height={ height }
+						imageSizeOptions={ imageSizeOptions }
+						isResizable={ isResizable }
+						imageWidth={ imageWidth }
+						imageHeight={ imageHeight }
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<InspectorAdvancedControls>
+				<TextControl
+					label={ __( 'Title attribute' ) }
+					value={ title || '' }
+					onChange={ onSetTitle }
+					help={
+						<>
+							{ __(
+								'Describe the role of this image on the page.'
+							) }
+							<ExternalLink href="https://www.w3.org/TR/html52/dom.html#the-title-attribute">
+								{ __(
+									'(Note: many devices and browsers do not display this text.)'
+								) }
+							</ExternalLink>
+						</>
+					}
+				/>
+			</InspectorAdvancedControls>
+		</>
+	);
+
+	// Disable reason: Each block can be selected by clicking on it
```

Comment:
```suggestion
	// Disable reason: Each block can be selected by clicking on it.
```

Reply:
These are all existing lines that I've moved around.

Reply:
I just figured we could fix this typo while we're at it, but I'm okay if we leave it for now if we want to keep this PR as minimal as possible.

Reply:
Oh, I see you just committed it, haha. :laughing: